internal/dao/dao_interfaces.go

```diff
@@ -14,11 +43,20 @@ type AccountDao interface {
 	List(ctx context.Context, limit, offset uint64) ([]*models.Account, error)
 }
 
-var GetPubkeyDao func(ctx context.Context) (PubkeyDao, error)
+var GetPubkeyDao func(ctx context.Context, tx Transaction) (PubkeyDao, error)
```

Comment:
All `GetXXXDao` functions now require a tx interface, which is typically sqlx.DB but it can be also sqlx.Tx for explicit transactions.

Reply:
Could this have a default value? I am bit worried this creates more boilerplate then necessary, but if it is easy to create such Transaction object to use, I'm ok.

Reply:
It could only be a second function. I think it is not worth the effort, `db.DB` will always be named this way even if we change its type.